circle.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
   pre:
     - sudo apt-get update; sudo apt-get -y install python-pip libglew-dev gdb
     - sudo pip install awscli
+  override:
+    - npm update && npm install
```

Comment:
Are both `update` and `install` required? [The docs](https://docs.npmjs.com/cli/update) say update "will also install missing packages".